Python/Module.cpp

```diff
@@ -631,12 +649,33 @@ PyMODINIT_FUNC initPyHSPlasma() {
     PyModule_AddObject(module, "plShadowMaster", Init_pyShadowMaster_Type());
     PyModule_AddObject(module, "plPointShadowMaster", Init_pyPointShadowMaster_Type());
     PyModule_AddObject(module, "plDirectShadowMaster", Init_pyDirectShadowMaster_Type());
-    PyModule_AddObject(module, "pfGUICtrlProcObject", Init_pyGUICtrlProcObject_Type());
-    PyModule_AddObject(module, "pfGUIDialogProc", Init_pyGUIDialogProc_Type());
-    PyModule_AddObject(module, "pfGUICtrlProcWriteableObject", Init_pyGUICtrlProcWriteableObject_Type());
+    PyModule_AddObject(module, "pfGUIButtonMod", Init_pyGUIButtonMod_Type());
+    PyModule_AddObject(module, "pfGUICheckBoxCtrl", Init_pyGUICheckBoxCtrl_Type());
+    PyModule_AddObject(module, "pfGUIClickMapCtrl", Init_pyGUIClickMapCtrl_Type());
```

Comment:
Using `PyType_CheckAndReady` will help ensure these are initialized in the right order (bases must be initialized before subtypes) -- currently they are not, and will cause methods and properties to be missing at runtime.